### PR TITLE
Created 3 basic test cases for Events, Food, and Friends pages; fixed minor typo in Food page

### PR DESCRIPTION
**Pull request by tbg510**
## Issue Number
#58 Expand test coverage to 90%
## Description
Created 3 test cases that just render the page content of the Events, Food, and Friends pages. They then check if the expected content is what's rendered in the test. Just made 3 test cases for now in case others wanted to create some.
Also fixed a minor typo in the Food page (Todays Menu -> Today's Menu).
## Testing
All test cases made in this pull request pass since the page content is just static text.
## Possible Errors
Test cases could be written in a better way, if so I'm open to suggestions for edits. When these pages get edited in the future the test cases will have to be edited as well to match any changes.
## Images 
image
image

Updated test coverage as of this pull request

### CODE REVIEW

**Review by junxjr — APPROVED** [review]
The three test cases looks good to me. Since the pages only contains text for now, only those can be tested. Great work!

File: small-talk/app/food/page.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function food() {
             <div className="flex flex-row">
                 <ul className="m-8 w-full">
                     <li className="border-b border-gray-200 pb-2 mb-10">Order Food</li>
-                    <li className="border-b border-gray-200 pb-2 mb-10">Todays Menu</li>
+                    <li className="border-b border-gray-200 pb-2 mb-10">Today&apos;s Menu</li>
```
Comment (blake-gilmore) [inline small-talk/app/food/page.js:12]: Was this the error from the linter? Very weird

**Review by blake-gilmore — APPROVED** [review]
This helps get us that coverage. Something to keep in mind with these tests is that minor changes to the wording or structure of these pages will result in failed tests, which could be a bit of a pain to adjust. Though not exactly sure what the best practice is for simply checking pages are rendering properly.

**Review by aiharc1 — APPROVED** [review]
Nice work getting the test coverage up! Looks good to me

File: small-talk/app/food/page.js (hunk 1 above)
Comment (tbg510) [reply small-talk/app/food/page.js:12]: Yea, I originally changed it to "Today's Menu" and the jest test didn't give any errors/warnings, just the linter.

Comment (blake-gilmore) [reply small-talk/app/food/page.js:12]: Sounds like a pain. We could potentially just ignore that specific rule from the linter

**Review by jusasin — APPROVED** [review]
Great documentation and it adds good coverage for pages that should have basic tests for them since they are simply features.

**Pull request merged**